packages/block-library/src/categories/index.php

```diff
@@ -31,6 +31,7 @@ function render_block_core_categories( $attributes ) {
 		$wrapper_markup           = '<div class="%1$s">%2$s</div>';
 		$items_markup             = wp_dropdown_categories( $args );
 		$type                     = 'dropdown';
+		$title                    = __( 'Categories', 'gutenberg' );
```

Comment:
The `'gutenberg'` domain is no longer used for translation. The second argument should be omitted.

Reply:
@aduth `'gutenberg'` exists in a couple spots in here, should I just remove them all? Thanks

Reply:
I think it's that your branch has drifted from `master`, where there is currently no domain added for the "Select Category" translated text.

https://github.com/WordPress/gutenberg/blob/40d1282a992702e1933118ef626f0666692baf2b/packages/block-library/src/categories/index.php#L30

You'll need to either merge in or rebase your branch against master for this to be reflected.

Reply:
Yeah It's been a while since I've synced this up, so I merged in master and made this change. Thanks for the feedback. Is there a good way other people handle keeping their branches up to date especially when some PRs can span a few weeks?

Reply:
Personally I make extensive use of [`git rebase -i origin/master` and `git push -f`](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) both to keep the commit history clean (squash, fixup) and to keep the branch closely in sync with current `master`.

The only caution is that it can be disruptive if you have multiple people collaborating on a branch, but I find these conflicts rarely (never) occur.

Reply:
Thanks for the tip. I've made the changes, but for some reason I'm getting failed unit tests, and I can't seem to run the test script locally as it fails all the tests.

Reply:
> Thanks for the tip. I've made the changes, but for some reason I'm getting failed unit tests, and I can't seem to run the test script locally as it fails all the tests.

The build occasionally fails, which is unfortunate, but can be cleared up by restarting it from Travis. I've done so and the build is now once again passing.